packages/evolution-frontend/src/components/inputs/InputMapFindPlace.tsx

```diff
@@ -38,7 +38,7 @@ export type InputMapFindPlaceProps<CustomSurvey, CustomHousehold, CustomHome, Cu
     widgetConfig: InputMapFindPlaceType<CustomSurvey, CustomHousehold, CustomHome, CustomPerson>;
 };
 
-interface InputMapFindPlaceState {
+interface InputMapFindPlaceState<CustomSurvey, CustomHousehold, CustomHome, CustomPerson> {
```

Comment:
Does this interface need to be generic? The generic types don't appear to be used

Reply:
> Does this interface need to be generic? The generic types don't appear to be used

The question remains here